### PR TITLE
fix(configurable styles) correctly add them to the build tree

### PR DESCRIPTION
**Pull request by kybishop**
Bring #7 past the finish line

### CODE REVIEW

**Comment by kybishop** [comment]
Build failure can be ignored for now, it just has to do with node-sass not being rebuilt properly by Travis. Will fix this in an upcoming PR to bump all the deps.

File: index.js [hunk 1]
```diff
@@ -37,30 +32,19 @@ module.exports = {
...
-  _flexiConfig: null,
-  flexiConfig: function() {
-    if (!this._flexiConfig) {
-      this._flexiConfig = getValidatedFlexiConfig(this.project.root);
-    }
-
-    return this._flexiConfig;
+  treeForAddonStyles(tree) {
+    return mergeTrees([
+      tree,
```
Comment (kybishop) [inline index.js]: This appears to work correctly, though it isn't the same [approach used in the original PR](https://github.com/html-next/flexi-default-styles/pull/7/files#diff-168726dbe96b3ce427e7fedce31bb0bcR49).

**Comment by rwwagner90** [comment]
@kybishop if you change https://github.com/html-next/flexi-default-styles/blob/master/.travis.yml#L36 to `yarn install --force` it will fix the travis issue.

**Comment by kybishop** [comment]
Thanks @rwwagner90!

Going to merge this since "it works on my machine"™. We can revisit this if people report bugs

**Pull request merged**